MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -266,24 +273,21 @@ void vtkSlicerMarkupsToModelLogic::UpdateSelectionNode( vtkMRMLMarkupsToModelNod
 //------------------------------------------------------------------------------
 void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel(vtkMRMLMarkupsToModelNode* markupsToModelModuleNode)
 {
-  vtkMRMLMarkupsFiducialNode* markups = markupsToModelModuleNode->GetMarkupsNode(); 
+  vtkMRMLMarkupsFiducialNode* markups = markupsToModelModuleNode->GetMarkupsNode();
   if(markups == NULL)
   {
-    vtkWarningMacro("No markups yet");
     return;
   }
   int numberOfMarkups = markups->GetNumberOfFiducials();
 
-  if(numberOfMarkups< MINIMUM_MARKUPS_CLOSED_SURFACE_NUMBER)
+  if(numberOfMarkups < 1)
   {
-    //vtkWarningMacro("Not enough fiducials for closed surface");
```

Comment:
The warning was already commented-out, I just removed the line. It is no longer necessary at all, because now even one fiducial is enough to draw something. Previously, it would not even attempt to draw a model if there were less than 4 points, because then they must be on the same plane.